manager/integration/tests/test_backing_image.py

```diff
@@ -511,6 +513,40 @@ def test_backing_image_auto_resync(client, volume_name):  # NOQA
     6. Wait for the file recovering automatically.
     7. Validate the volume.
     """
+    # Step 1
+    create_backing_image_with_matching_url(
+              client, BACKING_IMAGE_NAME, bi_url)
+
+    # Step 2
+    volume = create_and_check_volume(
+                                     client, volume_name, 3,
+                                     str(BACKING_IMAGE_EXT4_SIZE),
+                                     BACKING_IMAGE_NAME)
+
+    # Step 3
+    lht_host_id = get_self_host_id()
+    volume.attach(hostId=lht_host_id)
+    volume = wait_for_volume_healthy(client, volume_name)
+    assert volume.backingImage == BACKING_IMAGE_NAME
+    assert volume.size == str(BACKING_IMAGE_EXT4_SIZE)
+
+    # Step 4
+    subprocess.check_output(['rm', '-rf', '/var/lib/longhorn/backing-images/'])
+
+    # Step 5
+    disk_uuid = get_disk_uuid()
+    wait_for_backing_image_fail(client, BACKING_IMAGE_NAME, disk_uuid)
```

Comment:
NIT: `wait_for_backing_image_in_disk_fail`

Reply:
HI @shuo-wu , I update the function name, thank you